mmv1/provider/terraform/examples.rb

```diff
@@ -181,6 +181,38 @@ def config_documentation(pwd)
               ))
       end
 
+      def config_cloud_docs(pwd)
+        docs_defaults = {
```

Comment:
I couldn't figure out how to add `docs_defaults` as a global or class variable while making rubocop happy. Any tips or should I just let the duplicate from `config_documentation` exist

Reply:
You could return it from a function? It's fine to copy also, imo.